manifest.md

```diff
@@ -0,0 +1,261 @@
+<!--[metadata]>
++++
+draft = true
++++
+<![end-metadata]-->
+
+# Image Manifest
+
+The first goal is content-addressable images, by supporting an image model where the image's configuration can be hashed to generate an ID for the image and its components.
+The second goal is to allow multi-architecture images, through a "fat manifest" which references image manifests for platform-specific versions of an image.
+The third goal is to be translatable to the [OpenContainers/runtime-spec](https://github.com/opencontainers/runtime-spec)
+
+# Media Types
+
+The following media types are used by the manifest formats described here, and the resources they reference:
+
+- `application/vnd.oci.image.manifest.v1+json`: Image manifest format
```

Comment:
If we are starting a fork, should there not be a base commit before changing mediatypes? Do we intend to do a full compatibility break with Docker V2.2?

It might be appropriate to declare a mapping of equivalent mediatypes from OCI. For example V2.2 becomes, V1 of OCi and it is compatible with `application/vnd.docker...`.